projects/opendr_ws/src/perception/scripts/rgbd_hand_gesture_recognition.py

```diff
@@ -103,8 +103,8 @@ def preprocess(self, image, depth_img):
         :param depth_data: input depth image
         :type depth_data: engine.data.Image
         '''
-        image = image.numpy() / (2**8 - 1)
-        depth_img = depth_img.numpy() / (2**16 - 1)
+        image = image.convert(format='channels_last') / (2**8 - 1)
```

Comment:
Should we also add here channel_order="bgr" since we feed this image into cv2.resize in l.110?

Reply:
I am not sure what you mean? cv2.resize does not change the channel ordering, so to my understanding this is fine

Reply:
You are correct! It is indeed fine.